Rakefile

```diff
@@ -1,7 +1,9 @@
+# frozen_string_literal: true
```

Comment:
I think these should go away, we either do it everywhere or nowhere.

Reply:
Rubocop disagrees. What would you prefer?

Reply:
Actually it stopped complaining about it after running with a `--auto-gen-config` config, so this should be good now.

Reply:
I'm ok with it.

Personally, I like these to be consistent. So I generally don't have the `#frozen` in there and disable it in Rubocop or `--auto-gen-config`. This was added in this diff, that's why I am calling it out. You could revert this file and re-run `--auto-gen-config` and it would be better.